development/cpp/configuring_an_ide.rst

```diff
@@ -263,6 +263,60 @@ Test it:
 - It should break at the point!
 
 
+.. _doc_configuring_an_ide_vs:
+
+Visual Studio
+--------------
+
+Visual Studio Community is free for use and has lots of great features, such as memory view, performance view, source control and more.
```

Comment:
```suggestion
Visual Studio Community is free for non-commercial use and has lots of great features, such as memory view, performance view, source control and more.
```

(It can be free for commercial use in some cases, but let's keep it simple.)